src/content/config.ts

```diff
@@ -19,7 +19,17 @@ const work = defineCollection({
     }),
 });
 
+const blog = defineCollection({
+    type: "content",
+    schema: z.object({
```

Comment:
I would add meta of created at of ISO-8601 format with UTC+0.
So we can render posts in order.
Also, we should consider adding authors.